packages/dashboard/src/utils/grid.ts

```diff
@@ -8,6 +8,11 @@ export async function getGrid(mnemonic: string) {
     mnemonic,
     network: config.network as NetworkEnv,
     backendStorageType: BackendStorageType.tfkvstore,
+    relayURL: config.relay,
+    graphqlURL: config.graphqlUrl,
+    substrateURL: config.wsUrl,
+    activationURL: config.activationServiceUrl,
+    proxyURL: config.gridproxyUrl,
```

Comment:
We only need to set URLs in case the network is custom, other than that, no need to. This goes for all changes made.

Reply:
done